apps/koillection/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '3'
+
+services:
+    koillection:
+        image: koillection/koillection:1.4.11
+        container_name: koillection
+        restart: unless-stopped
+        ports:
+            - ${APP_PORT}:80
+        depends_on:
+            - db
+        volumes:
+            - ${APP_DATA_DIR}/data/volumes/koillection/uploads:/uploads
+        environment:
+            - APP_DEBUG=0
+            - APP_ENV=prod
+            - HTTPS_ENABLED={HTTPS_ENABLED}
+            - UPLOAD_MAX_FILESIZE=20M
+            - PHP_MEMORY_LIMIT=512M
+            - PHP_TZ=${TZ}
+            - CORS_ALLOW_ORIGIN='^https?://(localhost|127\.0\.0\.1)(:[0-9]+)?$'
```

Comment:
This regex only allows request from `localhost` not sure this will work with a domain exposed setup